core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -675,6 +679,20 @@ public static int defaultMaxTotalAttempts() {
         return ROUTE_CACHE_SPEC;
     }
 
+    /**
+     * Returns the value of the {@code routeDecoratorCache} parameter. It would be used to create a Caffeine
+     * {@link Cache} instance using {@link Caffeine#from(String)} mapping a route to decorator.
```

Comment:
I know this is just copied from similar flags, but maybe it's a good time to provide pointers on what the format of a spec is. I notice the `Caffeine#from` link just goes to the top of the page for `Caffeine`, and even the from method javadoc doesn't provide much. We should provide a link to [CaffeineSpec](https://static.javadoc.io/com.github.ben-manes.caffeine/caffeine/2.8.0/com/github/benmanes/caffeine/cache/CaffeineSpec.html). We might also want to give examples.

Reply:
@anuraaga That is a good idea. I will update the doc. 👍